route_following_plugin/src/route_following_plugin.cpp

```diff
@@ -96,26 +96,30 @@ namespace route_following_plugin
 
         double target_speed=findSpeedLimit(current_lanelet);   //get Speed Limit
 
-        //double total_maneuver_length = current_progress + mvr_duration_ * target_speed; // TODO STOPPING_ISSUE we can do a min of this line and the line below. 
-        double total_maneuver_length = wm_->routeTrackPos(shortest_path.back().centerline2d().back()).downtrack; // TODO STOPPING_ISSUE Maneuver extends to the end of the route???
+        double total_maneuver_length = current_progress + mvr_duration_ * target_speed; // TODO STOPPING_ISSUE we can do a min of this line and the line below. 
```

Comment:
This is a bit simplistic since the speed limit could change before the 15s maneuver duration has expired. But we can leave that change till later since vanden-plas only has one strategic plugin.